src/utils/src/net/mac.rs

```diff
@@ -104,6 +118,7 @@ impl MacAddr {
     /// let mac = MacAddr::from_bytes(&[0x01, 0x02, 0x03, 0x04, 0x05, 0x06]).unwrap();
     /// println!("{}", mac.to_string());
     /// ```
+    /// TODO this is unused, do we need it?
```

Comment:
Good catch. I would say remove it as a separate commit.

Reply:
If we don't want to check the length, then we have a `from_bytes_unchecked`

Reply:
Yes I understand that there's another function.
I'm trying to make sure that we have no use case where we'll need to verify length. 

If there are none, we can remove it.

Reply:
well if we need to verify then I think it is better to implement `TryFrom` trait

Reply:
This was meant as an interface, i say we leave it as it is